internal/policy/policy.go

```diff
@@ -223,6 +223,55 @@ func GetStateForCommit(ctx context.Context, repo *git.Repository, commit *object
 	return LoadStateForEntry(ctx, repo, commitPolicyEntry)
 }
 
+// Keys returns all the public keys associated with a state.
+func (s *State) Keys() (map[string]*tuf.Key, error) {
```

Comment:
nit: do we want to call this `PublicKeys` so there's no ambiguity for what kind of keys this is returning?